packages/vue-components/src/panels/NestedPanel.vue

```diff
@@ -92,11 +92,13 @@
             :fragment="fragment"
             @src-loaded="retrieverUpdateMaxHeight"
           />
-          <panel-switch
-            v-show="isExpandableCard && bottomSwitchBool"
-            :is-open="localExpanded"
-            @click.native.stop.prevent="toggle(true)"
-          />
+          <div class="card-bottom">
```

Comment:
Maybe `bottom-button-wrapper` is a better class name? Its similar to what we call our button-wrapper class now :)